packages/blade/src/components/Counter/_decisions/decisions.md

```diff
@@ -0,0 +1,46 @@
+# Counter 
+
+Counters are visual indicators that contains numerical values, tallies or counts in regards to some context.
+
+<img src="./counter-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8222%3A70827) to all variants of the Counter component
+
+## API
+
+| Prop | Type | Default | Description | Required |
+|---|---|---|---|---|
+| value | `number` | `undefined` | The numeric value of the counter which will be shown. | ✅ |
+| max | `number` | `undefined` | The max value of the counter, if the `value` exceededs `max` then the counter will show a `+` at the end. |  |
```

Comment:
can we make it more verbose? `max` right now seems vague of what it will refer to? maybe `maxValue` or something better you can think of?

Reply:
Wanted to keep things concise like `<input type="number" value="10" max="5" />` 

Tho don't have any strong opinions in this, `maxValue` also sounds okay.

Reply:
makes sense. i think let it be `max`